raiden/transfer/utils.py

```diff
@@ -8,6 +8,10 @@ def hash_balance_data(
         locked_amount: typing.TokenAmount,
         locksroot: typing.Locksroot,
 ) -> str:
+    if all([transferred_amount == 0, locked_amount == 0, locksroot == b'']):
```

Comment:
Please just use `transferred_amount == 0 and locked_amount == 0 and locksroot == b''`

`all([])` will create a temporary list and do a function call for something that is supported in the language.